.editorconfig

```
@@ -3,6 +3,7 @@ root = true
 
 [*]
 indent_style = tab
+indent_size = 4
```

Looks like `2` is the most common value now ;-)

```suggestion
indent_size = 2
```